torch/distributions/bernoulli.py

```diff
@@ -25,27 +27,29 @@ class Bernoulli(Distribution):
 
     def __init__(self, probs):
         self.probs, = broadcast_all(probs)
-
-    def sample(self, sample_shape=()):
-        if len(sample_shape) == 0:
-            return torch.bernoulli(self.probs)
-        elif len(sample_shape) == 1:
-            return torch.bernoulli(self.probs.expand(sample_shape[0], *self.probs.size()))
+        if isinstance(probs, Number):
+            batch_shape = ()
```

Comment:
nit: let's use `torch.Size` to be consistent with types

Reply:
Good catch, will change.